### ISSUE TITLE
nist-nestor install version on numpy not working on Mac with M1

### ISSUE THREAD

**Issue by sarahlukens** [usnistgov/nestor#4]
Hi Thurston - I tried a pip install nist-nestor and get an error with numpy - a "legacy-install-failure".  From reading the internet, this appears to be very common when installing numpy 1.16 on a Mac with M1, and in fact, I ran a pip install numpy==1.16 to check and got the same identical error as nist-nestor.   It looks like nestor is using 1.17.3, how do I specify 1.21.4 (or >1.19.4) ?

**Comment by rtbs-dev** [comment]
Hi Sarah! Sorry for the delay, things have been a bit hectic to dive into issue triage :sweat_smile: 

I believe this (and possibly #5 as well?) should be addressed already in `dev`, which is [pending a merge for v0.5](https://github.com/usnistgov/nestor/pull/2) Could you try to install this version directly? 

```bash
pip install git+https://github.com/usnistgov/nestor.git@dev
```

We're planning to release v0.5 to pypi shortly, to reflect the [current docs](https://pages.nist.gov/nestor/). Just needed a few more ci/cd changes to automate things before that, and we've been a bit busy internally with some major overhauls for v1.0

**Comment by rtbs-dev** [comment]
This should be resolved for v0.5
